src/back/auth/src/service/tokens_internal.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <string>
+
+namespace svetit::auth::tokens {
+
+class InternalTokens final {
```

Comment:
Слово tokens есть в названии namespace
```suggestion
class Internal final {
```

Reply:
поправил.